src/common/lib/client/realtimechannel.ts

```diff
@@ -149,11 +149,12 @@ class RealtimeChannel extends EventEmitter {
     return this._push;
   }
 
+  /** @spec RTL27 */
```

Comment:
I think that this notation is new to the codebase (elsewhere we just mention spec point identifiers without any special notation). Is there any particular reasoning behind the notation?